sphinx/roles.py

```diff
@@ -269,10 +266,6 @@ def run(self) -> tuple[list[Node], list[system_message]]:
         return super().run()
 
 
-_litvar_re = re.compile('{([^}]+)}')
-parens_re = re.compile(r'(\\*{|\\*})')
```

Comment:
Technically, it's not really the same regex so I don't know why it was changed in the class var. Could you check if this is used on github?

Reply:
Don't see any uses 😅 https://github.com/search?q=parens_re+language%3APython&type=code&p=2&l=Python

Reply:
Good, then we can remove it (and honestly, if anyone used it they half-play with fire IMO).